### ISSUE TITLE
Create user with multiple roles

### ISSUE THREAD

**Issue by damian-hanashiro** [headyj/terraform-provider-neo4j#6]
I'm trying to create a new user with multiple roles, like this:

resource "neo4j_user" "ruser" {
  name     = "my_user"
  password = "my_password"
  roles    = ["publisher", "editor"]
}

But I'm getting this error:

│ Error: Neo4jError: Neo.ClientError.Statement.ArgumentError (Failed to grant role 'editor,publisher' to user 'my_user': Role does not exist.)

I understand that the "roles" variable is a set of strings. Am I using it incorrectly?

**Comment by headyj** [comment]
You have to create both a role and an associated user. In your case it would look like this:
```
resource "neo4j_role" "my_role" {
    name = "myrole"
}
resource "neo4j_user" "ruser" {
    name = "my_user"
    password = "my_password"
    roles = [neo4j_role.my_role.name]
}
```
and then give the role one or multiple grants (with `neo4j_grant` resource) to access the resources like graph or databases

**Comment by brandocomando** [comment]
I'm seen this error as well. seems like its setting just one of the roles to the user.

**Comment by headyj** [comment]
@brandocomando yes you're right there's a bug with multiple role. I will do a fix

**Comment by headyj** [comment]
fixed by [df5a2a1766cafaa37715b8ae066eda280849fb39](https://github.com/headyj/terraform-provider-neo4j/commit/df5a2a1766cafaa37715b8ae066eda280849fb39)
